lib/src/components/NxTabs/__tests__/NxTabs.test.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import NxTabs from '../NxTabs';
+
+describe('NxTabs', function () {
+  it('renders NxTabs', function () {
+    const component = shallow(<NxTabs activeTab="activeTabId"/>);
+
+    expect(component).toHaveValue('activeTabId');
+    expect(component.dive()).toHaveClassName('nx-tabs');
+  });
```

Comment:
Needs tests that the activeTab gets passed down to the children appropriately (via Context)

Reply:
That's what the `toHaveValue` test is for (needed changes now that the context is a bit more complex). I'm checking that the context provider has the correct value set to pass down to the children.

Reply:
Lots of changes to this test
6fe2aeb